docs/source-app/workflows/mount_cloud_object_store.rst

```diff
@@ -0,0 +1,150 @@
+:orphan:
+
+################
+Mount Cloud Data
+################
+
+**Audience:** Users who want to read files stored in a Cloud Object Bucket in an app.
+
+******************************
+Mounting Public AWS S3 Buckets
+******************************
+
+===================
+Add Mount to a Work
+===================
+
+To mount data from a cloud bucket to your app compute, initialize a :class:`~lightning_app.storage.mount.Mount`
+object with the source path of the s3 bucket and the absolute directory path where it should be mounted and
+pass the :class:`~lightning_app.storage.mount.Mount` to the :class:`~lightning_app.utilities.packaging.cloud_compute.CloudCompute`
+of the :class:`~lightning_app.core.work.LightningWork` it should be mounted on.
+
+In this example, we will mount an S3 bucket: ``s3://ryft-public-sample-data/esRedditJson/`` to ``/content/esRedditJson/``.
+
+.. code:: python
+    :emphasize-lines: 9-12
+
+    import lightning as L
+    from lightning_app import CloudCompute
+    from lightning_app.storage import Mount
+
+    class Flow(L.LightningFlow):
+       def __init__(self):
+           super().__init__()
+           self.my_work = MyWorkClass(
+               cloud_compute=CloudCompute(
```

Comment:
Only need to show CloudCompute there, no need for the entire flow and work.

Reply:
I agree, let's only keep 
```
           self.my_work = MyWorkClass(
               cloud_compute=CloudCompute(
                   mounts=Mount(
                       source="s3://ryft-public-sample-data/esRedditJson/",
                       mount_path="/content/esRedditJson/",